### PR TITLE
Pirmasens toilet app

### PR DESCRIPTION
**Pull request by qhouyee**
First version of the Pirmasens Toilet App

Note that no development has been made in a while, and this PR is to consolidate existing work into the main branch, so that new users can further extend this in future if necessary.

### CODE REVIEW

File: Apps/PirmasensToiletApp/feature/routing/src/main/res/drawable/detail.png
Comment (sandradeng20) [inline Apps/PirmasensToiletApp/feature/routing/src/main/res/drawable/detail.png:1]: this file is used in `Apps/PirmasensToiletApp/feature/routing/src/main/res/layout/fragment_map.xml` as an invisible placeholder. This and the invisible component in the layout file should be removed together.

File: Apps/PirmasensToiletApp/feature/routing/src/main/res/drawable/walk.png
Comment (sandradeng20) [inline Apps/PirmasensToiletApp/feature/routing/src/main/res/drawable/walk.png:1]: not used, should be removed

File: Apps/PirmasensToiletApp/feature/routing/src/main/res/layout/fragment_map.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    <ImageView
+        android:id="@+id/imageView"
+        android:layout_width="0dp"
+        android:layout_height="257dp"
+        android:src="@drawable/detail"
+        android:visibility="invisible"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent" />
```
Comment (sandradeng20) [inline Apps/PirmasensToiletApp/feature/routing/src/main/res/layout/fragment_map.xml:29]: remove this as well as the `Apps/PirmasensToiletApp/feature/routing/src/main/res/drawable/detail.png`

File: Apps/PirmasensToiletApp/stack-data-uploader-inputs/README.md
Comment (sandradeng20) [inline Apps/PirmasensToiletApp/stack-data-uploader-inputs/README.md:1]: should files in `stack-data-uploader-inputs\` be placed in dropbox or somewhere else?

File: Apps/PirmasensToiletApp/stack-manager-inputs/inputs/data/webspace/dtvf/dtvf.min.css
Comment (sandradeng20) [inline Apps/PirmasensToiletApp/stack-manager-inputs/inputs/data/webspace/dtvf/dtvf.min.css:1]: having files duplication from dtvf/ and a lot images in `PirmasensToiletApp/stack-manager-inputs/`. please clean up

Comment (qhouyee) [reply Apps/PirmasensToiletApp/stack-manager-inputs/inputs/data/webspace/dtvf/dtvf.min.css:1]: Removed and clean it up in commit 43b0cbd9a7a2fc3448f21e1134638831270375bd

File: Apps/PirmasensToiletApp/stack-data-uploader-inputs/README.md
Comment (qhouyee) [reply Apps/PirmasensToiletApp/stack-data-uploader-inputs/README.md:1]: Reorganised the file structure. At the moment, these files are required queries and should be left on the github repo. Actual data will be stored elsewhere and be given only when access is requested.

File: Apps/PirmasensToiletApp/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+## Mapbox token
+
+Your mapbox token should be in [`settings.gradle`](./gradle.properties) and also [`developer-config.xml`](./core/utils/src/main/res/values/developer-config.xml). 
+
+## Phone and Stack communication
+If you are running on emulator use `HOST_LOCALHOST` in [`NetworkConfiguration.java`](./core/network/src/main/java/uk/ac/cam/cares/jps/network/NetworkConfiguration.java), otherwise you should have android and stack on same network. You can use `ipconfig` or other related commands to find the ip address of your pc.
+
+
+## Uploading the Data
+Place the relevant files from stack-data-uploader-inputs and stack-manager-inputs to stack-data-uploader and stack-manager folders respectively.
```
[the remark this answers is not in the document]
Comment (qhouyee) [reply Apps/PirmasensToiletApp/README.md]: Done

**Review by sandradeng20 — APPROVED** [review]

**Pull request merged**